src/components/TodoList/TodoList.scss

```diff
@@ -0,0 +1,46 @@
+.item-enter {
```

Comment:
Make sure to follow the naming conventions for block and element names by using underscores to separate them. It seems that the current class names do not follow this convention.